fe1-web/res/img/delete.svg

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
```

Comment:
did you draw this yourself or did you get it from somewhere?
If it's the latter, you might want to reference the source and include the license terms in the project.

Reply:
The author of the icon is Google.
https://www.flaticon.com/packs/action
Where should I include that source?

Reply:
Ah yes perfect. This seems to be the original source indeed.
Here it is: https://github.com/google/material-design-icons/blob/master/src/action/delete/materialicons/24px.svg

Reply:
If that's the original, you might want to use it as-is here :)